### PR TITLE
feat(plugin): BetterJoinedDate

### PR DESCRIPTION
**Pull request by AutumnVN**
image

### CODE REVIEW

File: src/plugins/betterJoinedDate.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { Tooltip } from "@webpack/common";
+
+export default definePlugin({
+    name: "BetterJoinedDate",
+    authors: [Devs.AutumnVN],
+    description: "Add a tooltip to the joined date showing the exact time and how many days ago it was",
+    patches: [{
```
Comment (Vendicated) [inline src/plugins/betterJoinedDate.tsx:15]: both of these matches are pretty bad. `children:(\(.+stuff` means find the first children:( in the file, then match all characters up to the first instance of stuff. the odds of this matching the wrong thing are incredibly high

Comment (AutumnVN) [reply src/plugins/betterJoinedDate.tsx:15]: ok i changed it

**Comment by UlyssesZh** [comment]
I didn't see this pr, but I implemented this again in #1784.

**Comment by Vendicated** [comment]
hmm i think it'd be better to format the date exactly like Discord's own timestamps
grafik

i think they use moment for that

**Pull request closed without merging**